### PR TITLE
`isLoading` codemod

### PR DESCRIPTION
**Pull request by balazsmatepetro**

### CODE REVIEW

File: packages/codemods/src/v5/is-loading/is-loading.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  const root = jscodeshift(file.source)
+  const utils = createUtilsObject({ root, jscodeshift })
+  const filePath = file.path
+
+  const dependencies = { jscodeshift, utils, root, filePath }
+
+  transformUsages({
+    ...dependencies,
+    config: {
+      hooks: ['useQuery'],
```
Comment (balazsmatepetro) [inline packages/codemods/src/v5/is-loading/is-loading.js:195]: @TkDodo what other hooks are affected?

File: packages/codemods/src/v5/is-loading/__testfixtures__/default-import.input.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import * as React from 'react'
```
Comment (balazsmatepetro) [inline packages/codemods/src/v5/is-loading/__testfixtures__/default-import.input.tsx:1]: @TkDodo does it cover most of the use cases? Feel free to provide a few if you think it lacks some of possible use cases. :)

Comment (TkDodo) [reply packages/codemods/src/v5/is-loading/__testfixtures__/default-import.input.tsx:1]: 👍

File: packages/codemods/src/v5/is-loading/is-loading.js (hunk 1 above)
Comment (TkDodo) [reply packages/codemods/src/v5/is-loading/is-loading.js:195]: `useMutation` is also affected

Comment (balazsmatepetro) [reply packages/codemods/src/v5/is-loading/is-loading.js]: https://github.com/TanStack/query/pull/6388/commits/66035671e34e06c064e2acfdcc3fb4136988f933

**Pull request merged**